release/src/main/scripts/build_release_candidate.sh

```diff
@@ -20,7 +20,8 @@
 # 1. Build and stage java artifacts
 # 2. Stage source release on dist.apache.org
 # 3. Stage python binaries
-# 4. Create a PR to update beam-site
+# 4. Stage SDK docker images
```

Comment:
Do we need to verify the staging images like what we did on the release candidates?

Reply:
The images will be verified in two steps.
1. Tests running by `verify_release_build.sh` will test images. Jenkins build images and use them for testings. Though the tests don't directly use staging images, the images are build from the exact same codebase by calling the same gradle command. 
2. For staging images, we should validate them in the same way we verify other artifacts, when we ask for votes from community.

Reply:
SG. Remember to update the template in the release sign-off sheet if staging image validation is required.